server/http/handlers/options.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	"github.com/xichen2020/eventdb/parser/json"
 	"github.com/xichen2020/eventdb/parser/json/value"
-	"github.com/xichen2020/eventdb/x/safe"
+	"github.com/xichen2020/eventdb/x/unsafe"
```

Comment:
use safe?

Reply:
The usage of unsafe here is self contained as well.

```
 // defaultTimeNanosFn parses the time value as a string in RFC3339 format.
 func defaultTimeNanosFn(v *value.Value) (int64, error) {
-       str, err := v.String()
+       b, err := v.Bytes()
        if err != nil {
                return 0, err
        }
-       t, err := time.Parse(time.RFC3339, str)
+       t, err := time.Parse(time.RFC3339, unsafe.ToString(b))
        if err != nil {
                return 0, err
        }
```

Reply:
Agreed that the usage is self contained but let's still use safe here. We should restrict usage of unsafe to a bare minimum (guideline being we should not use it unless the usage is simple, self-contained, and absolutely critical to performance).